packages/jetpack-core/composer.json

```diff
@@ -0,0 +1,40 @@
+{
+	"name": "automattic/jetpack-core",
```

Comment:
Can we name this Jetpack Bridge?

Now that A8c design is starting to work on emotional design, bridge is a more emotional word than core. Core is a more technical and developer-ish term than bridge, and can equally refer to the Earth's core or the center portion of a fruit.

On the other hand, bridge is a human construction, a social instrument, an element to bring spaces together. People walk on a bridge, not so much on a core. A bridge communicates.

Bridge is good because it also hints the communication between a plugin/feature/module/piece in the ecosystem and WPCOM. In this sense, it's self-explanatory about its role.

Reply:
Unfortunately bridge has specific technical connotations in multiple domains (networking, message passing in/out of VMs, etc), and "core" is widely understood in the developer community to mean "the base, required part of a system of libraries". I would also find "common" to be acceptable, though that is often used to describe a utility library, not the library that actually boots up a system.

99% of end users will never see this, and developers will be less confused about what a "core" (or "common") library is than what a "bridge" is. I am all for emotional design at the user facing level, and trying to use the most common terms for things at a developer-facing level.